Source/Core/DolphinQt/EmulatedUSB/Keyboard.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright 2025 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "DolphinQt/EmulatedUSB/Keyboard.h"
+
+#include <QCheckBox>
+#include <QComboBox>
+#include <QGridLayout>
+#include <QGroupBox>
+#include <QHBoxLayout>
+#include <QLabel>
+#include <QVBoxLayout>
+
+#include "Common/Keyboard.h"
+#include "Core/Config/MainSettings.h"
+#include "DolphinQt/Settings.h"
+
+KeyboardWindow::KeyboardWindow(QWidget* parent) : QWidget(parent)
+{
+  // i18n: Window for managing the Wii keyboard emulation
+  setWindowTitle(tr("Keyboard Manager"));
+  setObjectName(QStringLiteral("keyboard_manager"));
+  setMinimumSize(QSize(500, 200));
+
+  auto* main_layout = new QVBoxLayout();
+
+  {
+    auto* group = new QGroupBox();
+    auto* hbox_layout = new QHBoxLayout();
+    hbox_layout->setAlignment(Qt::AlignHCenter);
+    auto* checkbox_emulate = new QCheckBox(tr("Emulate USB keyboard"), this);
+    checkbox_emulate->setChecked(Settings::Instance().IsUSBKeyboardConnected());
+    connect(checkbox_emulate, &QCheckBox::toggled, this, &KeyboardWindow::EmulateKeyboard);
+    connect(&Settings::Instance(), &Settings::USBKeyboardConnectionChanged, checkbox_emulate,
+            &QCheckBox::setChecked);
+    hbox_layout->addWidget(checkbox_emulate);
+    group->setLayout(hbox_layout);
+
+    main_layout->addWidget(group);
+  }
+
+  {
+    auto* group = new QGroupBox(tr("Layout Configuration"));
+    auto* grid_layout = new QGridLayout();
+    auto* checkbox_translate = new QCheckBox(tr("Enable partial translation"), this);
```

Comment:
We have a `ConfigBool` subclass of `QCheckBox` that will automatically do loading and saving of config.